packages/ui/src/Button/ButtonDefault.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { Button, ButtonProps } from "./Button";
+
+export const ButtonDefault = (props: ButtonProps) => {
```

Comment:
I would add a `@deprecated` comment to the components too.

This is the comment I use:


```
@deprecated This component is deprecated and will be removed in future releases. 
Please use the `<Button />` component from the `@webiny/admin-ui` package instead.
```

Reply:
Added.